src/main/kotlin/com/launchdarkly/intellij/LaunchDarklyApiClient.kt

```diff
@@ -33,12 +33,13 @@ class LaunchDarklyApiClient() {
         @JvmStatic
         fun projectInstance(project: Project?, apiKey: String? = null, baseUri: String? = null): ProjectsApi {
             val settings =
-                if (project != null) LaunchDarklyMergedSettings.getInstance(project) else LaunchDarklyApplicationConfig.getInstance().ldState
+                if (project != null) LaunchDarklyApplicationConfig.getInstance().ldState else LaunchDarklyApplicationConfig.getInstance().ldState
```

Comment:
it won't let me suggest, bit we don't need this if anymore - unless you want to return early? not sure that makes sesne